tests/X86/SSE/PSHUFHW.S

```diff
@@ -0,0 +1,30 @@
+#define PSHUFHW_INPUTS_64 \
+    0xF82DAC26, 0x091D41A7, \
+    0x5FD4D3D9, 0x39A5f1D8, \
+    0x063F0DDF, 0xAA9352B2, \
+    0xD4A19DBB, 0x2D81A606, \
+    0xBD2A7045, 0x8FC05637, \
+    0x75D9B52A, 0x0AA1300B, \
+    0x55535FB8, 0x4952FFC7, \
+    0x06EA7DD6, 0x2D6EDF79, \
+    0x49974EBE, 0xD64D34E8, \
+    0x21E9840E, 0x6B56224B, \
+    0x5B3631FD, 0xA03337FF, \
+    0xEE92c9F3, 0x3B786526
+TEST_BEGIN_64(PSHUFHWv128v128imm8_6d, 2)
+TEST_INPUTS(PSHUFHW_INPUTS_64)
+    movq xmm0, ARG1_64
+    movq xmm1, ARG2_64
+    pshufhw xmm0, xmm1, 0x6d
+TEST_END_64
+
+TEST_BEGIN_64(SHUFPSv128v128imm8_b1, 2)
+TEST_INPUTS(PSHUFHW_INPUTS_64)
+    push ARG1_64
+    push ARG1_64
+    movq xmm0, [rsp]
+    push ARG2_64
+    push ARG2_64
+    movq xmm1, [rsp]
+    pshufhw xmm0, xmm1, 0xb1
+TEST_END_64
```

Comment:
Can you add some another tests that uses the default values in the `xmm` registers and exercises more selectors?

Reply:
@pgoodman Sure, I will do it.